parquet-fuzz/README.md

```diff
@@ -0,0 +1,20 @@
+# Fuzzing
```

Comment:
Can we rename the directory to just `fuzz` - having `parquet-` seems unnecessary.

Reply:
done